src/element/bounds.ts

```diff
@@ -152,6 +152,10 @@ const getCubicBezierCurveBound = (
   return [minX, minY, maxX, maxY];
 };
 
+const getRandomOffset = () => {
+  return Math.random() / 1_000_000;
```

Comment:
```suggestion
  return Math.random() / 1000000;
```

https://caniuse.com/mdn-javascript_grammar_numeric_separators isn't "universally" supported yet